projects/observability/src/shared/components/gauge/gauge.component.ts

```diff
@@ -8,11 +8,7 @@ import { Arc, arc, DefaultArcObject } from 'd3-shape';
     <div class="gauge-container" (htLayoutChange)="this.onLayoutChange()">
       <svg class="gauge" *ngIf="this.rendererData">
         <g attr.transform="translate({{ rendererData.origin.x }}, {{ rendererData.origin.y }})">
-          <path
-            class="gauge-ring"
-            [attr.d]="rendererData.backgroundArc"
-            *ngIf="rendererData.radius > ${GaugeComponent.GAUGE_MIN_RADIUS_TO_SHOW_PATH}"
-          />
+          <path class="gauge-ring" [attr.d]="rendererData.backgroundArc" *ngIf="rendererData.radius > 80" />
```

Comment:
Should this still be `GAUGE_MIN_RADIUS_TO_SHOW_PATH` ? Where did the hardcoded 80 come from?

Reply:
It was previously developed. It's used as an edge to allow render the corresponding arc, there is a relation between the arc and this radius. I can apply the change but probably @anandtiwary has a deeper understanding of this specific constant.

Reply:
let me cherry-pick and see.

Reply:
radius check was there previously, but in this PR we've changed it from `rendererData.radius > GaugeComponent.GAUGE_MIN_RADIUS_TO_SHOW_PATH` to `rendererData.radius > 80` - that change, more than the check itself is what I'm wondering about.

Reply:
I see. I will keep the constant. Thanks!